pkg/storage/replica_application_cmd.go

```diff
@@ -112,10 +118,10 @@ func (c *replicatedCmd) Rejected() bool {
 
 // FinishAndAckOutcome implements the apply.AppliedCommand interface.
 func (c *replicatedCmd) FinishAndAckOutcome() error {
```

Comment:
It's documented that this must only be called once, right?

Reply:
It is in https://github.com/cockroachdb/cockroach/pull/39468.